### ISSUE TITLE
Inconsistent code style

### ISSUE THREAD

**Issue by lipkau** [RamblingCookieMonster/BuildHelpers#74]
The functions in this repo have many different styles of code.

This is not a big problem, but a contributor using an editor that "fixes" the style (eg: VSCode), will flag a lot more changes than required.

Is it in the interest of this module to harmonize the style to avoid this? And if so, with what rules? Shall this be enforced with tests?

**Comment by RamblingCookieMonster** [comment]
Agreed, consistency would be good.  Will get back to you but might be a bit (other priorities on this and other projects, summit proposal pestering, wrapping up work before parental leave, etc.).

**Comment by pauby** [comment]
If we could get agreement on a primary style, PRs would be easier to submit?

**Comment by pauby** [comment]
> Not really
>* enable enforcing of common styling

... Which makes submitting PRs easier (saves the submitter having to guess which style to pick) 👍

**Comment by pauby** [comment]
> This helps with other stuff... EOL char, file encoding, indention, etc...

It does. It's limited. But using it stops you mandating people use VS Code (as believe it or not, not everybody does) and raising the bar for people becoming involved. For those people it's the only thing you can give them. That's why I suggested using both:

> Or use both (as they are unlikely to change much).

**Comment by lipkau** [comment]
> That's why I suggested using both

that's what I do in my modules :-)

**Comment by lipkau** [comment]
I would recommend **Stroustrup**

references:
https://en.wikipedia.org/wiki/Indentation_style#Styles
https://github.com/PoshCode/PowerShellPracticeAndStyle/issues/81#issuecomment-285835361

**Comment by lipkau** [comment]
If this can be defined, I can submit a PR by the next day :-)

**Comment by lipkau** [comment]
sample:
https://github.com/lipkau/BuildHelpers/tree/fix/harmonizeCodeStyle

diff:
https://github.com/RamblingCookieMonster/BuildHelpers/compare/master...lipkau:fix/harmonizeCodeStyle?expand=1

**Comment by lipkau** [comment]
Any news on this?
